crates/nu-cli/src/commands/commandline.rs

```diff
@@ -19,6 +19,7 @@ impl Command for Commandline {
             .input_output_types(vec![
                 (Type::Nothing, Type::Nothing),
                 (Type::String, Type::String),
+                (Type::String, Type::Int),
```

Comment:
Do we now allow divergence in output type? Otherwise this may still confuse the typecheck

Reply:
It seems to work but I guess we could have a separate subcommand like `commandline cursor`.

Reply:
I noticed that typecheck does not work well as @sholderbach pointed out.
```nu
❯ def foo []: string -> int { commandline -c }
Error: nu::parser::output_type_mismatch

  × Command output doesn't match int.
   ╭─[entry #2:1:29]
 1 │ def foo []: string -> int { commandline -c }
   ·                             ───────┬──────
   ·                                    ╰── command doesn't output int
   ╰────
```

It might be better to have a separate subcommand as @fdncred mentioned.
Should we revert this?

Reply:
We could revert it, or just add a new sub command and do the revert in that PR.

Reply:
I will create a PR to add a subcommand and I revert this in that PR.

Reply:
ok, sounds good. thanks.